radar/tools/pm2.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "PM2"
+ring: trial
```

Comment:
We have been using it in production for months and we have a PR open to make it default